### DISCUSSION TITLE
Wizard in symfony ux

### DISCUSSION THREAD

**Discussion by Ali-Kent** [symfony/ux#2478] [Questions & Answers] [answered]
Hello,

I would like to develop a wizard for editing quotes. I wanted to know if it was possible to do this with symfony ux.
If this is possible can you give me an idea to start?
Attached is a photo of the wizard.
![wizard](https://github.com/user-attachments/assets/d312519e-66bc-46cc-9234-8c5c5f416ada)
Thanks

**Comment by smnandre** [comment, marked as answer]
Hi @Ali-Kent: there is an ongoing MR/discussion https://github.com/symfony/ux/pull/2433

**Reply by Ali-Kent** [reply]
Thank you for your response.
This is a good starting point.
